src/lib/walletgen.js

```diff
@@ -15,14 +16,16 @@ import {
 
 const SEED_LENGTH_BYTES = SEED_ENTROPY_BITS / 8;
 
+const getEntropyBits = point =>
```

Comment:
Perhaps pedantic, but I'd `getTicketBitSize`, since we use it to determine that, not the "entropy" that goes in.

Reply:
teach me the dark arts of naming things good

Reply:
[Use this forbidden txt wisely.](https://github.com/urbit/urbit/blob/master/doc/spec/flw.txt)